articles/puppet-module.md

```diff
@@ -20,9 +20,9 @@ Install [Fleet's Puppet module](https://forge.puppet.com/modules/fleetdm/fleetdm
 
 ### Step 2: configure Puppet to talk to Fleet using Heira
 
-1. In Fleet, create an API-only user with the GitOps role. Instructions for creating an API-only user are [here](./fleetctl-CLI.md#create-an-api-only-user).
+1. In Fleet, create an API-only user with the GitOps role. Instructions for creating an API-only user are [here](https://fleetdm.com/guides/fleetctl#create-api-only-user).
```

Comment:
It seems that a bunch of relative links in the docs were broken at some point. From what I can see, it looks like it was probably when various files were renamed and shuffled from `docs/using-fleet` into `articles` (aka `guides`). 

I can see that there was some effort to define [redirects](https://github.com/fleetdm/fleet/blob/76e1862bf69be8ad299b8e0e599293f4344bc888/website/config/routes.js#L461-L462) on the website, but it doesn't appear to be capturing everything. My suspicion is that it works well enough with simple path changes (a -> b) but starts to break down when there are bigger changes. In this case, the relative link gets transformed to `fleetdm.com/articles/fleetctl-cli#create-an-api-only-user` but it none of the redirect rules are applied and the user ends up in 404-land.

In this PR, I swapped out all the relative links to markdown files for absolute links to the fleetdm.com site. I'm not sure how product/digital experience wants to resolve cases like this. Should we be using relative markdown links at all? If not, I think we ought to open a follow up ticket to fix relative markdown links and potentially add some kind of linter rule to watch out for them in the future.

Reply:
@gillespi314, thanks for jumping on this. We should only be using absolute links. I'll create a ticket for Digital Experience to check for stray relative links.